extensions/amp-ad-network-doubleclick-impl/0.1/doubleclick-a4a-config.js

```diff
@@ -122,12 +123,21 @@ export class DoubleclickA4aEligibility {
         !this.supportsCrypto(win)) {
       return false;
     }
+    const urlExperimentId = extractUrlExperimentId(win, element);
     let experimentName = DFP_CANONICAL_FF_EXPERIMENT_NAME;
     if (!this.isCdnProxy(win)) {
-      experimentId = this.maybeSelectExperiment_(win, element, [
-        DOUBLECLICK_EXPERIMENT_FEATURE.CANONICAL_CONTROL,
-        DOUBLECLICK_EXPERIMENT_FEATURE.CANONICAL_EXPERIMENT,
-      ], DFP_CANONICAL_FF_EXPERIMENT_NAME);
+      // Ensure that forcing FF via url is applied if test/localDev.
+      experimentId = urlExperimentId == -1 &&
```

Comment:
Add parens around the predicate, was a little confused reading this

Reply:
Done